Master cannot compile

```go
Building Milvus ...
# github.com/milvus-io/milvus/internal/proxy
internal/proxy/task.go:51:2: trace redeclared as imported package name
	previous declaration at internal/proxy/task.go:30:2
Makefile:92: recipe for target 'milvus' failed
make: *** [milvus] Error 2
```

--- a/.gitignore
+++ b/.gitignore
@@ -81,6 +81,7 @@ coverage.txt
 profile.out
 profile.txt
 go_coverage.html
+cover.out
 
 # codecov-cpp generated files
 lcov_*.info

--- a/internal/proxy/task.go
+++ b/internal/proxy/task.go
@@ -27,8 +27,6 @@ import (
 	"time"
 	"unsafe"
 
-	"github.com/milvus-io/milvus/internal/util/trace"
-
 	"github.com/milvus-io/milvus/internal/common"
 
 	"go.uber.org/zap"

